packages/plugin/src/generators/app/templates/webpack.config.js

```diff
@@ -70,5 +71,11 @@ module.exports = {
         new CopyWebpackPlugin({
             patterns: ['manifest.json'],
         }),
+        new ProvidePlugin({
+            process: 'process/browser',
```

Comment:
Is this to provide `process` in the browser?  Why is that needed?

Reply:
Yes, this resolves the issue of not being able to access environment variables using process.env.<<var>>

Reply:
Is there a scenario where we need process.env?  My feeling is there shouldn't be a need for using process.env anywhere in widgets.

Reply:
Currently there isn't. I just thought it'll be useful to include support for it incase they need something for their local code development outside of what the context lib is providing. And they needn't have to fiddle with the configuration.

Reply:
Well, the reason process/browser even exists (btw I assume for this to work we'd need to include 'process' as a dependency in package.json) is because there is no `process` or `process.env` in the browser.  Even if you include this, there wouldn't be anything in `process.env` when running in the browser, so I'm not sure what this would be used for.  If this is stemming from things like `STORYBOOK_*` environment variables, those are used at build time so it wouldn't need this plugin anyway.

Reply:
That's what I'm trying to think through..., they _could_ add things in process.env (yes, I agree `process` dependency will need to be added) for their code -- we can't predict what will be there in the widgets component code. (I guess I;m not sure that we can safely assume that there will _never_ be anything in the `process.env` when running in the browser)

Reply:
You can be sure there will never be anything in `process.env` because `process` doesn't even exist in the browser.  This code is polyfilling a global `process`, but there will be nothing in `process.env`, it'll be empty.  There's really no reason for widgets to be using `process`, and if for some reason someone does want a global `process` variable in the browser, it'd really be up to them to set that up.

> Also -- im curious -- when you mentioned STORYBOOK-* vars are used at build time -- won't those be a part of process.env?

If you mean at runtime in the browser, no.  There is no `process.env` in the browser.  If you try to use it, you'll get a 'process is undefined' error.  The webpack build replaces the `process.env.*` references to whatever value is in the environment at the time of the build.  The output of webpack doesn't have `process.env`.